Sync/BookmarkPayload.swift

```diff
@@ -352,7 +352,7 @@ open class BookmarkPayload: BookmarkBasePayload {
     }()
 
     lazy var tagsString: String = {
-        if self["tags"].type == Type.array {
+        if self["tags"].isArray() {
```

Comment:
just checking the `isArray()` function in `JSONExtensions.swift` (and all the other `is...` functions), these also don't check for `nil`, just for type. Is that the behaviour we want here, or should we add a `nil` check there too?